### PR TITLE
[CSL-1580] Run demo with 4 nodes by default

### PR DESCRIPTION
**Pull request by int-index**

### CODE REVIEW

File: scripts/launch/demo.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ if [[ ! -d "$base/../../.stack-work" ]]; then
 fi
 
 # Define the default amount of nodes to run
-DEFAULT_NODES_N=3
+DEFAULT_NODES_N=4
```
Comment (volhovM) [inline scripts/launch/demo.sh]: can you throw an error if `N /= 4` ? Just in case, because it doesn't work otherwise.

**Comment by volhovM** [comment]
Only script-related changes, merging.

File: scripts/launch/demo.sh (hunk 1 above)
Comment (gromakovsky) [reply scripts/launch/demo.sh]: What exactly doesn't work?

File: scripts/launch/demo.sh [hunk 2]
```diff
@@ -17,14 +17,19 @@ if [[ ! -d "$base/../../.stack-work" ]]; then
...
 
 # Detect node number or use default
 n=$1
 if [[ "$n" == "" ]]; then
   n=$DEFAULT_NODES_N
 fi
 
+if [[ "$n" != 4 ]]; then
+  echo "Currently we don't support any amount of nodes in demo that's not 4. See CSL-1580"
```
Comment (gromakovsky) [inline scripts/launch/demo.sh]: Weird :/

File: scripts/launch/demo.sh [hunk 3]
```diff
@@ -17,14 +17,19 @@ if [[ ! -d "$base/../../.stack-work" ]]; then
...
+  exit
```
Comment (gromakovsky) [inline scripts/launch/demo.sh]: Revert it please. It shouldn't exit in this case.

Comment (volhovM) [reply scripts/launch/demo.sh]: ok

Comment (volhovM) [reply scripts/launch/demo.sh]: `76874b40f4ff723cc5274fe88a2acd0ba7cb877f`

**Pull request merged**